org.lflang/src/org/lflang/generator/ts/TSGenerator.kt

```diff
@@ -213,10 +207,15 @@ class TSGenerator(
      * Copy the TypeScript runtime so that it is accessible to the generated code.
      */
     private fun copyRuntime() {
-        for (runtimeFile in RUNTIME_FILES) {
+        FileUtil.copyDirectoryFromClassPath(
+            "$LIB_PATH/reactor-ts/src/core",
```

Comment:
Next to `core` we have `benchmark` and `share`, which seemed to be separate from the core functionality of the runtime. I guessed that the runtime itself was supposed to be limited to `core`. Those directories are small, however, so it would be harmless to include them.

Reply:
Ah, right. I think what you did makes sense then. Let's keep it as it!